src/DependencyInjection/Compiler/ConfigParserPass.php

```diff
@@ -212,10 +213,12 @@ function (array $typeMapping) use ($container) {
     private function mappingFromBundles(ContainerBuilder $container): array
     {
         $typesMappings = [];
+
+        /** @var array<string, class-string> $bundles */
         $bundles = $container->getParameter('kernel.bundles');
 
         // auto detect from bundle
-        foreach ($bundles as $name => $class) {
+        foreach ($bundles as $class) {
             // skip this bundle
             if (OverblogGraphQLBundle::class === $class) {
```

Comment:
Why this change in this PR? Please open a dedicated PR for this.

Reply:
https://github.com/overblog/GraphQLBundle/pull/816